source/blog/2018-05-18-the-emberjs-times-issue-47.md

```diff
@@ -0,0 +1,136 @@
+
+---
+
+## [Meta and Link all the Things in Ember Data 🐹](https://github.com/emberjs/rfcs/pull/332)
+
+This backwards compatible Request for Comments (RFC) allows `ember-data` users to associate optional `meta` and `links` on **records** alongside the top-level document `meta` (mandatory) and `links` (optional).  Sadly, `ember-data` currently stores this information only on relationships and not individual records.  
+
+The RFC design is pretty straightforward and proposes that all `meta` and `links`:
+
+- will be honored in any resource that is part of a [JSON API compliant](http://jsonapi.org/) document
+- when defined on resource identifiers (e.g. links inside a relationhip that link to other resources) will be ignored
+- when defined on relationship objects will continue to function as they do today
+- will be exposed as getters on instances of `DS.Model` and will default to null if nothing has been provided
+
+Similarly, `meta` and `links` will be exposed on instances of `DS.Snapshot`. Although `meta` and `links` are getters on `DS.Model`s, they will be exposed on `DS.Snapshot` as methods:
+
+```js
+class Snapshot {
+    links() {}
+	meta() {}
+}
+```
+
+Interestingly, some apps have inadvertently achieved what this RFC purposes by moving `meta` and `links` into `attributes` during serialization and then exposing each of them via `DS.attr`. 
+Therefore, this RFC gives `ember-data` an even more **complete "out of the box experience"** and will likely be a very welcomed addition to the library.  
+
+Think differently? Tell [@runspired](https://github.com/runspired) what you think on the [original proposal](https://github.com/emberjs/rfcs/pull/332).
+
+---
+
+## [Only 12 more days of #EmberJS2018 ✍️💨](https://emberjs.com/blog/2018/05/02/ember-2018-roadmap-call-for-posts.html)
+
+12 months in a year. 12 eggs in a dozen. 12 days to submit your blog posts for the [#EmberJS2018 Roadmap RFC](https://emberjs.com/blog/2018/05/02/ember-2018-roadmap-call-for-posts.html)! 📝
+
+Need to beat writer's block? Listen to Ember Core team member [Chad Hietala](https://github.com/chadhietala) on The Changelog podcast, discussing [Ember four years later](https://changelog.com/podcast/293). And you can peruse blog posts from the community on [@zinyado's repo](https://github.com/zinyando/emberjs2018-posts) or the [#EmberJS2018 hashtag](https://twitter.com/search?q=%23EmberJS2018) on Twitter. Get your two cents in **by May 30th**.
+
+---
+
+## [More Ember Data: RFC for removing `Ember.Evented`](https://github.com/emberjs/rfcs/pull/329)
+
+There's an [RFC](https://github.com/emberjs/rfcs/pull/329) on removing usage of the `Ember.Evented` mixin in Ember Data specifically. This would also lead to the future **removal** of several **lifecycle hooks** and methods on `Model`s and other Ember Data classes. 
```

Comment:
remove first hyperlink of `[RFC]` since it's already linked in the title and in `[RFC pull request.]`?

change `[RFC pull request.]` to `[RFC pull request].`